components/List/index.js

```diff
@@ -3,9 +3,21 @@ import Link from "next/link";
 import Image from "next/image";
 
 export default function List({ players }) {
-    return (
-      <StyledList>
-        {players.map((player) => {
+  return (
+    <StyledList>
+      {players
+        .sort((a, b) => {
+          const nameA = a.player.lastname.toLowerCase();
+          const nameB = b.player.lastname.toLowerCase();
+          if (nameA < nameB) {
+            return -1;
+          }
+          if (nameA > nameB) {
+            return 1;
+          }
+          return 0;
+        })
```

Comment:
I would outsource the comparator outside the component as:

```js
function comparePlayers(a, b) {
  const nameA = a.player.lastname.toLowerCase();
  const nameB = b.player.lastname.toLowerCase();
  if (nameA < nameB) {
    return -1;
  }
  if (nameA > nameB) {
    return 1;
  }
  return 0;
};
```

Then in JSX just `{players.sort(comparePlayers).map(...)}`